README.md

```diff
@@ -79,7 +89,68 @@ instances:
     ssl: True
 ```
 
-During the Dyno start up, your YAML files will be copied to the appropriate Datadog Agent configuration directories.
+During the Dyno start up, your YAML files is copied to the appropriate Datadog Agent configuration directories.
+
+## Heroku Log Collection
```

Comment:
@l0k0ms The buildpack doesnt collect logs, so adding this portion is confusing. Let's separate the logs doc out and link to it from here.

Reply:
fix here: https://github.com/DataDog/heroku-buildpack-datadog/pull/95